modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/index.js

```diff
@@ -16,6 +16,23 @@ import {fetch} from 'frontend-js-web';
 
 import createOdataFilter from './odata';
 
+export function getAcceptLanguageHeaderParam() {
+	const browserLang = navigator.language || navigator.userLanguage;
+	const themeLang = Liferay.ThemeDisplay.getLanguageId().replace('_', '-');
+
+	if (browserLang === themeLang) {
+		return browserLang;
+	}
+
+	return `${browserLang}, ${themeLang};q=0.8`;
```

Comment:
Why exactly the 0.8? Do we have user data to back this decision?

Reply:
Hi Chema! I checked the Chrome default ones and I replicated them here, no other reasons

Reply:
Sounds good enough @FabioDiegoMastrorilli!!